### PR TITLE
Add stress test framework

### PR DESCRIPTION
**Pull request by Eli-Meitner**

### CODE REVIEW

**Review by ShakedYosef — APPROVED** [review]

File: modules/test-framework-manager/lambda/test_framework_manager.js [hunk 1]
```diff
@@ -18,19 +18,18 @@ let env_name;
...
 exports.handler = function (event, context, callback) {
   console.log('event', event);
   if (event.DeploymentType == "AppMesh") {
-      deploymentType = "AppMesh";
-      taskToken = event.TaskToken;
+    deploymentType = "AppMesh";
+    taskToken = event.TaskToken;
   }
-  //if (event.hasOwnProperty("DeploymentId")) {
   else if (event.DeploymentId) {
```
Comment (srinivas-toluna) [inline modules/test-framework-manager/lambda/test_framework_manager.js:29]: you wanted to keep the prop check with hasOwnProperty() in today discussion, but you removed it. Any specific reason?

Comment (Eli-Meitner) [reply modules/test-framework-manager/lambda/test_framework_manager.js:29]: I took a chance to check if it is still needed and it worked without , so it is OK to remove

File: modules/test-framework-manager/lambda/test_framework_manager.js [hunk 2]
```diff
@@ -136,67 +130,71 @@ async function runIntegrationTest(app_config) {
...
-          deploymentId: `${deploymentId}`,
-          environment: `${app_config['CONFIG_DETAILS'].environment}`, 
-          report_group: `${app_config['REPORT_GROUPS'].stress_report_group_arn}`, 
-          lb_name: `${app_config['LB_NAME']}`,
-          port: port,
-          trigger: `${process.env.APP_NAME}-${process.env.ENV_TYPE}-test-framework-manager`,
-          DeploymentType : deploymentType, 
-          TaskToken: taskToken
+    Payload: JSON.stringify({
+      runIntegrationTests: runIntegrationTests,
```
Comment (srinivas-toluna) [inline modules/test-framework-manager/lambda/test_framework_manager.js:166]: why do we need runIntegrationTests info for runStressTests as part of payload?

**Review by srinivas-toluna — APPROVED** [review]
Approved,

**Pull request merged**